container/controller.go

```diff
@@ -693,7 +695,7 @@ func (c *Controller) checkPrereqs(prereqsPassed chan bool, rpcDead chan struct{}
 			if !failedAny {
 				glog.Infof("Passed all prereqs.")
 				prereqsPassed <- true
-				return nil
+				break WaitingForPrereqs
```

Comment:
Wait - is the purpose of this to break out of the for? Hmm. I guess I'm not a fan of this style - if I were reading it, it would take me longer to parse the break than the return.